string[] ingredient_names
--- 
Steps recipe